src/services/users/actions.js

```diff
@@ -69,3 +69,7 @@ export function updateUser(data) {
 export function setPageNumber(pageNumber) {
   return setPageN(DOMAIN, pageNumber)
 }
+
+export function goToEdit(id) {
+  return push(`/my-account/users/edit/${id}`)
+}
```

Comment:
What's the difference between using `push(href)` and using a `<Link to={href} />`?

If there's none, I'd much rather use the 2nd form directly in the component.